documentation/asciidoc/accessories/camera/camera_installing.adoc

```diff
@@ -14,41 +14,45 @@ Depending on the model, the camera may come with a small piece of translucent bl
 
 === Enabling the Camera
 
-==== Using the desktop
-
-Select `Preferences` and `Raspberry Pi Configuration` from the desktop menu: a window will appear. Select the `Interfaces` tab, then click on the `enable camera` option. Click `OK`. You will need to reboot for the changes to take effect.
-
-==== Using the command line
-
-Open the `raspi-config` tool from the terminal:
+Before proceeding, we recommend ensuring that your kernel, GPU firmware and applications are all up to date. You'll need an internet connection for this to work correctly.
 
 [,bash]
 ----
-sudo raspi-config
+sudo apt update
+sudo apt full-upgrade
 ----
 
-Select `Interfacing Options` then `Camera` and press `Enter`. Choose `Yes` then `Ok`. Go to `Finish` and you'll be prompted to reboot.
+==== Using `libcamera`
 
-=== Setting up the Camera Software
+Raspberry Pi's three official cameras (V1, V2 and HQ camera) will be detected and enabled automatically when using a Raspberry Pi OS based on _Bullseye_ or later.
 
-Execute the following instructions on the command line to download and install the latest kernel, GPU firmware, and applications. You'll need an internet connection for this to work correctly.
+If you are using a 3rd party camera, or wish to use a non-standard driver with an official Raspberry Pi camera, or are using an OS that is earlier than _Bullseye_, please refer to the xref:camera.adoc#getting-started[`libcamera` _Getting Started_ instructions].
+
+To check that everything is working correctly, enter:
 
 [,bash]
 ----
-sudo apt update
-sudo apt full-upgrade
+libcamera-hello
 ----
 
-Now you need to enable camera support using the `raspi-config` program you will have used when you first set up your Raspberry Pi.
+You should see a camera preview window for about 5 seconds.
+
+==== Using the legacy stack
+
+===== On the desktop
+
+Select `Preferences` and `Raspberry Pi Configuration` from the desktop menu: a window will appear. Select the `Interfaces` tab, then click on the `enable camera` option. Click `OK`. You will need to reboot for the changes to take effect.
```

Comment:
Perhaps "click on the `enable camera` option" should be "click on the `Camera: Enabled` option" ?